fflas-ffpack/utils/fflas_randommatrix.h

```diff
@@ -434,8 +434,10 @@ namespace FFPACK{
         RandomTriangularMatrix (F, N, N, FFLAS::FflasUpper, FFLAS::FflasNonUnit, true, U, N, G);
 
         typename Field::Element alpha, beta;
-        F.init(alpha,1.0);
-        F.init(beta,0.0);
+        F.init(alpha);
```

Comment:
Replace alpha,beta by F.one and F.zero in the call to fgemm below and remove them